lstchain/scripts/lstchain_data_create_drs4_pedestal_file.py

```diff
@@ -42,7 +42,7 @@
                     help="Path to fitz.fz file to create pedestal file.",
                     default=None, required=True)
 
-parser.add_argument("--output-file", '-o', , type=str, action='store',
+parser.add_argument("--output-file", '-o', type=str, action='store',
```

Comment:
a test should be able to catch this (I guess the `--help` test you are proposing will do it)

Reply:
This is how I found it!